hles/transformation/src/test/scala/org/broadinstitute/monster/dap/dog/DemographicsTransformationsSpec.scala

```diff
@@ -391,7 +391,7 @@ class DemographicsTransformationsSpec extends AnyFlatSpec with Matchers with Opt
     val example = Map[String, Array[String]](
       "dd_dog_weight" -> Array("3"),
       "dd_dog_weight_lbs" -> Array("100.25"),
-      "dd_weight_range_expected_adult" -> Array("4")
+      "dd_dog_weight_as_adult" -> Array("4")
```

Comment:
it looks like the ticket spells this as `dd_dog_weigh_as_adult`, with no 't' in 'weight'.

Reply:
@ThePletch The ticket is incorrect, grepping the latest extracted json shows data for `dd_dog_weight_as_adult`, and none for `dd_dog_weigh_as_adult`

Reply:
We've confirmed the `dd_dog_weight_as_adult` with the DAP team.